data/dev.js

```diff
@@ -135,3 +135,69 @@ export const people = [
     alt: 'acm logo',
   },
 ];
+
+export const projects = [
+  {
+    name: 'ACM at UCLA Website',
+    prim_lang: 'JS',
+    proj_link: 'https://www.uclaacm.com',
+    repo_link: 'https://github.com/uclaacm/website',
+    description: `The ACM at UCLA Website is the one-stop shop for all things ACM! We develop 
+    and maintain this website, keeping it up-to-date with all the latest news and events. `,
+    img: '/images/dev-projects/acmwebsite.png',
+    alt: 'ACM Website!',
+  },{
+    name: 'Membership Portal',
+    prim_lang: 'JS',
+    proj_link: 'https://members.uclaacm.com/login',
+    repo_link: 'https://github.com/uclaacm/membership-portal',
+    description: `The Membership Portal is where ACM members can check in to events to earn 
+    points! We built the Membership Portal with Node.js, Express.js, PostgreSQL, Redux and React.js.`,
+    img: '/images/dev-projects/membership.png',
+    alt: 'Membership Portal!',
+  },{
+    name: 'Open Source',
+    prim_lang: 'JS',
+    proj_link: 'https://opensource.uclaacm.com/',
+    repo_link: 'https://github.com/uclaacm/opensource',
+    description: `Everything we build is open source! In the spirit of open-source, anyone is welcome 
```

Comment:
can we change description to:
Open Source highlights all open source projects managed by [ACM at UCLA](https://github.com/uclaacm), accompanied by an activity feed showcasing recent contributions to [repositories under ACM at UCLA](https://github.com/orgs/uclaacm/repositories). Anyone is welcome to contribute, making it an open invitation to get involved and make an impact!

Reply:
Changed a bit to fit in the conventional space